README.rst

```diff
@@ -234,6 +234,25 @@ To install the package in editable mode, you can use:
 
     $ pip install -e .
 
+
+Testing
++++++++
+
+Install test dependencies by running:
+
+.. code:: sh
+
+    $ pip install pytest mock
+
+Then run the tests using:
+
+.. code:: sh
+
+    $ python -m pytest
+
+You will need to close manually the plots that are generated during the tests, unless you define the ``CI`` environment variable, which will disable the tests that generate plots.
```

Comment:
I have wondered to myself why the plots do not block the CI runs.  Thanks for the insight :)